### ISSUE TITLE
missing entries

### ISSUE THREAD

**Issue by boba1l0s2k9** [QCodefag/QCodefag.github.io#4]
Why are some of the seemingly valid posts missing?

See here:
https://github.com/QCodefag/QCodefag.github.io/commit/ac1039c5c1ed9d3c58c322471fbc86c4931648ec#diff-cd1bd648709ff6246fe5f92e4ddff547

**Comment by qntmpkts** [comment]
Related: #3.

Perhaps the author intends to leave them out for an unstated reason. The commit comments make it clear, however, that they belong there.

**Comment by boba1l0s2k9** [comment]
I have confirmed that the "Q FULL MAP PDF" from CBTS_Stream on reddit does indeed contain the posts that are missing from qcodefag.  This web-based presentation of the Q posts is very nice, but if posts are missing it makes it almost useless.  You can't make bread without all the breadcrumbs.

**Comment by qntmpkts** [comment]
As I stated, it is unacceptable. I cannot fathom why it is so difficult to reinstate the posts.

**Comment by qntmpkts** [comment]
I've reinstated the missing posts in [my version](https://qntmpkts.keybase.pub), in case it remains unobvious to the author.

**Comment by QCodefag** [comment]
This is fixed now with commit 77ba801b440c699706a09735a493fc2f21dda1c8
I fixed this a while ago, but forgot to push the fix.
The posts got deleted because the 8chan threads have become archived. My scraping algorithm could not deal with that.
Being archived also means that the pictures will get removed from the 8chan server (after a while).

So to fix that I downloaded all the pictures (from db-q.com) and have a local back-up. (so I fixed the missing image posted in 12916)
